client/src/api/projects.js

```diff
@@ -19,6 +19,25 @@ export const addProject = async (projectData) => {
   }
 };
 
+export const editProject = async (projectId, newProjectData) => {
+  const url = `/projects/${projectId}`;
+  const token = localStorage.getItem('jwtToken');
+  const authOptions = {
+    headers: {
+      Authorization: `Bearer ${token}`
+    }
+  };
```

Comment:
Isn't the token sent in the httpClient?

Reply:
Yeah but I had the issue where it was undefined at times and this was a more consistent solution to that issue. I will try to see how it works without it now once we settle the other PR merges that should come before this PR

Reply:
hmm ok